### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by fskreuz**
**Description of the pull request:**

Update deps, because the deps shield is red.

**Reviewers:**

Everyone.

### CODE REVIEW

**Comment by evs-chris** [comment]
I tried about 6 weeks ago and gave up after an hour or so of hair-pulling. I'd say try 'em one at a time to avoid the massive pile of fail. I think we may need to stick with an older gobble to avoid the fs watch terribleness with native modules.

File: .travis.yml [hunk 1]
```diff
@@ -4,8 +4,6 @@ node_js:
   - '6'
   - '5'
   - '4'
-  - '0.12'
```
Comment (evs-chris) [inline .travis.yml]: Every time I change one of the node tests, I forget that they aren't transpiled, so the CI fails on node < 4. So a bit :+1: on this :smile:

**Comment by evs-chris** [comment]
Good to go?

**Comment by fskreuz** [comment]
Yep. This will do for now.

Determined that updating Gobble and/or Rollup causes issues. Will
investigate later.

**Comment by fskreuz** [comment]
The move was just in time for the news
https://medium.com/@nodejs/node-js-v6-transitions-to-lts-be7f18c17159 :D

**Pull request merged**